test/testutil.py

```diff
@@ -140,6 +140,21 @@ def create_filesystem_customizations(rootfs: str):
     ]
 
 
+def pull_container(container_ref, target_arch=""):
+    if target_arch == "":
+        target_arch = platform.machine()
+
+    if target_arch not in ["x86_64", "amd64", "aarch64", "arm64", "s390x", "ppc64le"]:
+        raise RuntimeError(f"unknown host arch: {target_arch}")
+
+    output = subprocess.run([
+        "podman", "pull",
+        "--arch", target_arch,
+        container_ref,
+    ], check=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding="utf8")
```

Comment:
Why do we capture the output here? It seems we are not doing anything with it. Maybe just remove the stdout/stderr/encoding here?